src/content/products/deepsparse/community.mdx

```diff
@@ -160,30 +149,31 @@ qa_pipeline = Pipeline.create(
 my_name = qa_pipeline(question="What's my name?", context="My name is Snorlax")
 ```
 
-NLP Tutorials:
-- [Getting Started with Hugging Face Transformers 🤗](https://github.com/neuralmagic/deepsparse/tree/main/examples/huggingface-transformers)
+Refer also to [NLP and Computer Vision Tasks Supported](https://github.com/neuralmagic/deepsparse/tree/main/examples).
 
-Tasks Supported:
+For NLP tutorials, see [Getting Started with Hugging Face Transformers](https://github.com/neuralmagic/deepsparse/tree/main/examples/huggingface-transformers).
+
+Tasks supported are:
 - [Token Classification: Named Entity Recognition](https://neuralmagic.com/use-cases/sparse-named-entity-recognition/)
 - [Text Classification: Multi-Class](https://neuralmagic.com/use-cases/sparse-multi-class-text-classification/)
 - [Text Classification: Binary](https://neuralmagic.com/use-cases/sparse-binary-text-classification/)
 - [Text Classification: Sentiment Analysis](https://neuralmagic.com/use-cases/sparse-sentiment-analysis/)
 - [Question Answering](https://neuralmagic.com/use-cases/sparse-question-answering/)
 
-### 🦉 SparseZoo ONNX vs. Custom ONNX Models
+### SparseZoo ONNX vs. Custom ONNX Models
 
 DeepSparse can accept ONNX models from two sources:
 
-- **SparseZoo ONNX**: our open-source collection of sparse models available for download. [SparseZoo](https://github.com/neuralmagic/sparsezoo) hosts inference-optimized models, trained on repeatable sparsification recipes using state-of-the-art techniques from [SparseML](https://github.com/neuralmagic/sparseml).
+- **SparseZoo ONNX** is our open-source collection of sparse models available for download. [SparseZoo](https://github.com/neuralmagic/sparsezoo) hosts inference-optimized models, trained on repeatable sparsification recipes using state-of-the-art techniques from [SparseML](https://github.com/neuralmagic/sparseml).
```

Comment:
@Beth-Kosis since you are proposing the changes, and Rob is countering with more changes, please reconcile the changes. To do that, simply edit the suggested change (three dots to the right of "member" above...)